rspec-mode.el

```diff
@@ -752,15 +752,19 @@ or a cons (FILE . LINE), to run one example."
           (t (rspec-project-root (file-name-directory (directory-file-name directory)))))))
 
 (defun rspec--include-fg-syntax-methods-p ()
-  "Check whether FactoryGirl::Syntax::Methods is included in spec_helper."
+  "Check whether FactoryGirl::Syntax::Methods is included in rails_helper or spec_helper."
   (cl-case rspec-snippets-fg-syntax
     (full nil)
     (concise t)
     (t
-     (with-temp-buffer
-       (insert-file-contents
-        (concat (rspec-project-root) "spec/spec_helper.rb"))
-       (re-search-forward "include +FactoryGirl::Syntax::Methods" nil t)))))
+     (cl-find-if
+      (lambda (path)
+        (when file-exists-p path
+              (with-temp-buffer
+                (insert-file-contents path)
+                (re-search-forward "include +FactoryGirl::Syntax::Methods" nil t))))
+      '((expand-file-name "spec/rails_helper.rb" rspec-project-root)
```

Comment:
Have you tried running this? It won't work:

* `rspec-project-root` is a function, not a variable.
* The quote before the list will stop the elements from being evaluated (so they're strings, not forms). Instead of rectifying that, though, it will be better to move the `expand-file-name` call inside the lambda too (with a let-binding). Or let-bind `default-directory` there instead.